src/chat-handler.ts

```diff
@@ -78,10 +78,16 @@ export class ChatHandler extends ChatModel {
   }
 
   async sendMessage(message: INewMessage): Promise<boolean> {
+    const body = message.body;
+    if (body.startsWith('/clear')) {
+      // TODO: do we need a clear method?
+      this.messagesDeleted(0, this.messages.length);
+      return false;
+    }
```

Comment:
We should also clear the message history `this._history.messages`, to avoid sending cleared messages.